endpointmanager/pkg/nppesquerier/populatenpi_integration_test.go

```diff
@@ -56,8 +56,8 @@ func Test_ParseAndStoreNPIFile(t *testing.T) {
 		t.Errorf("Error Parsing NPI File: %s", err.Error())
 	}
 	// Assert expected number of orgs are parsed out of fixture file
-	if parsed_orgs != 14 {
-		t.Errorf("Expected number or parsed orgs to be %d, got %d", 3, parsed_orgs)
+	if parsed_orgs != 12 {
+		t.Errorf("Expected number or parsed orgs to be %d, got %d", 13, parsed_orgs)
```

Comment:
12?

Reply:
I had to change the test to take in a different npidata pfile without any entries that were not organizations (Entity type 2), since the populatenpiorgs.go file no longer checks if the entry is of Entity type 2, it just assumes it is since the new NPPES script parses out all the entries that are not Entity type 2. Therefore, this new npidata pfile has 12 entries that are of Entity type 2. I can add or remove entries if you think there should be more/less, but I basically replaced the. file this test is testing against so that it has all Organization entries.

Reply:
ah whoops, i was referring to the error statement which has "13" instead of "12", but i appreciate the explanation, and I think that's a totally reasonable number for the file!